modules/040-node-manager/hooks/machineclass_checksum_assign_test.go

```diff
@@ -941,12 +941,26 @@ func newCloudProviderAvailabilityChecker() func(tYpE string) {
 // getAvailableCloudProviderTypes returns all cloud providers
 // containing corresponding checksum template in cloud-providers directory.
 func getAvailableCloudProviderTypes() set.Set {
-	ptypes := set.New()
+	const modulesDirCE = "/deckhouse/modules"
+	const modulesDirEE = "/deckhouse/ee/modules"
+	const modulesDirFE = "/deckhouse/ee/fe/modules"
```

Comment:
same question

Reply:
I've inlined this constants in https://github.com/deckhouse/deckhouse/pull/3740/commits/5fef301bb025fabb248f6d4fab7fc49136fd8485.